EpikCord/abstract.py

```diff
@@ -138,8 +131,8 @@ async def typing(self) -> TypingContextManager:
 
 
 class BaseCommand:
-    def __init__(self, checks: List[Check] = []):
-        self.checks: List[Check] = checks
+    def __init__(self, checks: List[Check] = None):
```

Comment:
Shouldn't you mark this as Optional then?

Reply:
indeed